src/index.html

```diff
@@ -11,7 +11,162 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset>
+        <legend>Personal Information</legend>
+
+        <label class="pers-inf">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          required
+          autocomplete="off"
+          >
```

Comment:
check your code style according to the checklist
<img width="915" alt="image" src="https://user-images.githubusercontent.com/102411130/225856465-18c9cd27-4608-42db-972e-93a0f536d2a4.png">

Reply:
fix all such cases